tests/phpunit/includes/Core/Modules/Module_With_Settings_ContractTests.php

```diff
@@ -39,7 +39,10 @@ public function test_setting_registration() {
 		$testcase    = $this->get_testcase();
 		$settings    = $this->get_module_with_settings()->get_settings();
 		$option_name = constant( get_class( $settings ) . '::OPTION' );
-		unregister_setting( $option_name, $option_name );
+
+		if ( array_key_exists( $option_name, get_registered_settings() ) ) {
+			unregister_setting( $option_name, $option_name );
+		}
```

Comment:
Not sure why this wasn't causing a problem before but the `Analytics_4Test` was failing from this because its setting isn't registered during the plugin bootstrap due to its conditional registration and all feature flags being off by default at that point.